examples/Makefile

```diff
@@ -90,15 +66,7 @@ build/%.tgz: */%/ build/newest build/manifest
 	find "$<" $(IGNORE) -delete
 	tar -czf "$@" -C $$(dirname "$<") $$(basename "$<")
 
-test:
-	pytest -vv -s --durations=0 tests
-
 fmt:
 	make -C tests fmt
 	black .
 	isort .
-
-check:
```

Comment:
is it worth keeping these build steps for the examples that are remaining in `examples`? i get that we don't have tests for them, but maybe we should add them in the future?

Reply:
On second thought, it's definitely a mistake to remove `check`, since it checks formatting of every example.

I do, however, think that `examples/tests` is redundant. We use examples in either `e2e_tests/test` or `harness/tests` depending on if it's a unit/integration/capability test. Any tests that are actively maintained in `determined` should be serving CI or serving Docs, as a showcase of an important feature that we maintain.